spot-client/src/spot-tv/ui/components/exit-electron/ExitVerification.js

```diff
@@ -26,18 +104,65 @@ export class ExitVerification extends React.Component {
     render() {
         const { onCancel, onVerification, productName, t } = this.props;
 
-        return (
-            <StatusOverlay title = { t('appStatus.exitElectronConfirm', { productName }) }>
-                <Button
-                    appearance = 'subtle'
-                    onClick = { onCancel }>
-                    { t('buttons.cancel') }
-                </Button>
-                <Button onClick = { onVerification }>
-                    { t('buttons.confirm') }
-                </Button>
-            </StatusOverlay>
-        );
+        switch (this.state.viewToDisplay) {
+        case VIEW_EXIT_CONFIRMATION:
+            return (
+                <StatusOverlay
+                    qaId = 'exit-electron-confirmation'
+                    title = { t('appStatus.exitElectronConfirm', { productName }) }>
+                    <Button
+                        appearance = 'subtle'
+                        onClick = { onCancel }
+                        qaId = 'exit-electron-cancel' >
+                        { t('buttons.cancel') }
+                    </Button>
+                    <Button
+                        onClick = { onVerification }
+                        qaId = 'exit-electron-confirm' >
+                        { t('buttons.confirm') }
+                    </Button>
+                </StatusOverlay>
+            );
+        case VIEW_LOADING:
+            return (
+                <StatusOverlay
+                    qaId = 'exit-electron-loading'
+                    title = { t('appStatus.pleaseWait', { productName }) }>
```

Comment:
Maybe `productName` is unnecessary for this string.

Reply:
Oh yeah actually this argument is not used anymore 🤦‍♂